### ISSUE TITLE
post launch item - translate PO names and roles on cards and bios as well as headers on translated pages

### ISSUE THREAD

**Issue by bobpullin** [FordFoundation/Redesign2015#70]
![untranslated headers on ta page](https://cloud.githubusercontent.com/assets/12086465/10822515/ff9058e8-7e2d-11e5-8e9e-ccf3fbb54510.jpg)

**Comment by ttrevisan** [comment]
not sure about this request - is this resolved or would you like to see different titles on the cards than what render in English?

**Comment by bobpullin** [comment]
Yes, the goal is to avoid bouncing back and forth between Chinese and English above the address bar. Related to the screen shot above we would see "Thematic Areas in this Region" in Chinese. We can discuss the effort required to translate the "Grantmakers in China" header as well as the staff names, titles and location. That would be highly desirable.

**Comment by ttrevisan** [comment]
This could be worth discussing on our **Friday call ** in more detail. There seems to be some UX/IA involved.  If you'd like to discuss sooner, let me know and i can find a half hour slot where Peter, Angela, and I could discuss?
